kolibri_explore_plugin/assets/src/components/InstallContentModal.vue

```diff
@@ -0,0 +1,230 @@
+        const color2 = 'rgba(255, 255, 255, 0.2)';
+        const color1 = 'rgba(255, 255, 255, 0)';
+        return {
+          background:
+            'linear-gradient(90deg,' +
+            `${color1} 0%, ${color1} ${stop}%,` +
+            `${color2} ${stop}%, ${color2} 100%)`,
+        };
+      },
+    },
+    watch: {
+      visibleInstallContentModal() {
+        this.isModalVisible = this.visibleInstallContentModal;
+      },
+    },
+    beforeDestroy() {
+      clearTimeout(this.pollingId);
+    },
+    mounted() {
+      this.pollJobs();
+      ChannelResource.useContentCacheKey = false;
+    },
+    methods: {
+      downloadContent() {
+        this.downloading = true;
+        const url = urls['kolibri:kolibri_explore_plugin:endless_learning_collection']();
+        this.jobs = [];
+        axios
+          .post(url)
+          .then(() => {
+            this.pollJobs();
+          })
+          .catch(() => {
+            this.downloading = false;
+          });
+      },
+      pollJobs() {
+        clearTimeout(this.pollingId);
+
+        const url = urls['kolibri:kolibri_explore_plugin:endless_learning_collection']();
+        axios.get(url).then(({ data }) => {
+          this.jobs = data;
+          const completedJobs = this.jobs.filter(j => j.status === 'COMPLETED');
+          const completed = completedJobs.length;
+
+          if (completed > 0 && completed === this.jobs.length) {
+            // Download is completed
+            this.$store.commit('SET_SHOW_INSTALL_CONTENT', false);
+            ChannelResource.clearCache();
+            showChannels(this.$store);
+            this.downloading = false;
+          } else {
+            if (completed !== this.channelsDownloaded) {
+              this.channelsDownloaded = completed;
+              ChannelResource.clearCache();
+              showChannels(this.$store);
+            }
+
+            this.downloading = this.jobs.some(j => j.status !== 'COMPLETED');
+            if (this.downloading) {
+              this.pollingId = setTimeout(() => this.pollJobs(), 1500);
+            }
+          }
+        });
+      },
+      onHide() {
+        this.$store.commit('SET_SHOW_INSTALL_CONTENT', false);
+      },
+      showModal() {
+        this.$store.commit('SET_SHOW_INSTALL_CONTENT', true);
+      },
+    },
+  };
+
+</script>
+
+
+<style lang="scss" scoped>
+
+  @import '../styles';
+
+  .install-content {
+    position: fixed;
+    bottom: $spacer;
+    left: $spacer;
+    z-index: $zindex-fixed;
+    // Remove 2px border from the actual size to match the slidable cards row:
+    width: $circled-button-size + 4px;
+    height: $circled-button-size + 4px;
+    padding: 0;
+  }
+
+  .button-progress {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+  }
+
+  .fade-enter-active,
+  .fade-leave-active {
+    @include transition($transition-fade);
+  }
+  .fade-enter,
+  .fade-leave-to {
+    opacity: 0;
+  }
```

Comment:
Why are all these definitions needed for transitions? They happen automatically in the About modal.

Reply:
I've removed the `fade` css classes to use the default.